cypress/support/step_definitions/index.ts

```diff
@@ -84,6 +100,46 @@ Given('the URL {string} responds with', (url: string, yaml: string) => {
 
 // act
 
+When('I visit the {string} URL', function (path: string) {
```

Comment:
this seems a bit unrelated to the PR title (not related to localStorage)

Reply:
Oh yeah totally fair 👍 

I mentioned in the desc:

> I also combined visit.ts here. We used to have this separate for reasons, but we no longer need it to be separate. (ln:103 down is just the contents of the deleted file)

But happy to put this in another PR also, lmk.

Reply:
no need